src/couchapps/WMStatsErl/views/latestRequest/map.erl

```diff
@@ -5,9 +5,7 @@ fun({Doc}) ->
     <<"agent_request">> ->
       Workflow = couch_util:get_value(<<"workflow">>, Doc),
       AgentUrl = couch_util:get_value(<<"agent_url">>, Doc),
-      Id = couch_util:get_value(<<"_id">>, Doc),
-      Timestamp = couch_util:get_value(<<"timestamp">>, Doc),
-      Emit([Workflow, AgentUrl], {[{id,Id},{timestamp,Timestamp}]});
+      Emit([Workflow, AgentUrl], null);
```

Comment:
Is the `Workflow` really the workflow name here (blah)? Or it's the agent_url + workflow name (vocms0250.cern.ch-blah)?

Reply:
It is just workflow name.

Reply:
The only deference of the view is value part new view has null, old view has id (workflow + agentURL) and timestamp
```
new view

{"total_rows":15,"offset":0,"rows":[
{"id":"vocms0193.cern.ch-sryu_DQMHarvesting_RunWhitelist_syu_181129_170448_3645","key":["sryu_DQMHarvesting_RunWhitelist_syu_181129_170448_3645","vocms0193.cern.ch"],"value":null},

old view

{"rows":[
{"key":["amaltaro_Run2018A-v1-EGamma-17Sep2018_1024p1_181018_183329_3950","cmsgwms-submit7.fnal.gov"],"value":{"id":"cmsgwms-submit7.fnal.gov-amaltaro_Run2018A-v1-EGamma-17Sep2018_1024p1_181018_183329_3950","timestamp":1543533540}},

```